packages/expo-linear-gradient/android/src/main/java/expo/modules/lineargradient/LinearGradientModule.kt

```diff
@@ -29,6 +29,10 @@ class LinearGradientModule : Module() {
       Prop("borderRadii") { view, borderRadii: FloatArray? ->
         view.setBorderRadii(borderRadii ?: FloatArray(8) { 0f })
       }
+
+      Prop("ditherPaintAndroid") { view, ditherPaint: Boolean ->
+        view.setDitherPaint(ditherPaint)
+      }
```

Comment:
```suggestion
      Prop("dither") { view, shouldDither: Boolean? ->
        view.setDitherPaint(shouldDither ?: false)
      }
```
Removing the prop will cause null to be passed here. I see you are always passing something on the JS side but it's still better to handle this in native.